packages/vanilla-bundle/src/components/__tests__/slick-vanilla-grid.spec.ts

```diff
@@ -659,12 +659,12 @@ describe('Slick-Vanilla-Grid-Bundle Component instantiated via Constructor', ()
         const mockCollection = ['male', 'female'];
         const promise = new Promise(resolve => resolve(mockCollection));
         const mockColDefs = [{ id: 'gender', field: 'gender', editor: { model: Editors.text, collectionAsync: promise } }] as Column[];
-        const getColSpy = jest.spyOn(mockGrid, 'getColumns').mockReturnValue(mockColDefs);
+        //const getColSpy = jest.spyOn(mockGrid, 'getColumns').mockReturnValue(mockColDefs);
 
         component.columnDefinitions = mockColDefs;
 
         setTimeout(() => {
-          expect(getColSpy).toHaveBeenCalled();
+          //expect(getColSpy).toHaveBeenCalled();
```

Comment:
instead of commenting the unit test lines, can you remove all of the ones you commented out please

Reply:
Done, removed unnecessary test from getcolumn method

Reply:
thanks :)